common/pb/response_adjust_mod.proto

```diff
@@ -0,0 +1,7 @@
+import "response.proto";
+
+message Response_AdjustMod{
+	extend Response {
+		optional Response_AdjustMod ext = 1011;
```

Comment:
I think i may look into fixing the proto warnings that get thrown for every proto file.  If I can fix those ill fix the spacing at that time.